TestProject/TestProject/UnitTest1.cs

```diff
@@ -0,0 +1,32 @@
+using System.Threading;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace TestProject;
+
+public class Tests
+{
+    private IWebDriver driver;
+    private readonly string HOME_PAGE_URL = "https://vadimevteev.testrail.io/";
+    
+    [SetUp]
+    public void Start()
+    {
+        driver = new ChromeDriver();
+        driver.Manage().Window.Maximize();
+    }
```

Comment:
Lets create `DriverFactory.cs` in order to easily access needed driver (Chrome, Firefiox etc)
And define needed driver in `appsetting.json`